config/v0.3.0/config.go

```diff
@@ -188,3 +179,26 @@ func createTLSConfig(caCertFile *string, clientCertFile *string, clientKeyFile *
 	}
 	return tlsConfig, nil
 }
+
+// createHeadersConfig combines the two header config fields. Headers take precedence over headersList.
+func createHeadersConfig(headers []NameStringValuePair, headersList *string) (map[string]string, error) {
+	result := make(map[string]string)
+	if headersList != nil {
+		headerslist, err := baggage.Parse(*headersList)
```

Comment:
I understand the parsing comes from https://github.com/open-telemetry/opentelemetry-specification/blob/c65b2b6fe01cd52a3272bd9f91bae48c8057df69/specification/protocol/exporter.md#specifying-headers-via-environment-variables

However,  it is not specified anywhere in https://github.com/open-telemetry/opentelemetry-configuration. See: https://github.com/search?q=repo%3Aopen-telemetry%2Fopentelemetry-configuration%20headers_list&type=code

I think this is OK for this, but https://github.com/open-telemetry/opentelemetry-configuration probably should be updated. @open-telemetry/configuration-maintainers, WDYT?

Reply:
The configuration schema has a reference to the specification (https://github.com/open-telemetry/opentelemetry-configuration/blob/568e5080816d40d75792eb754fc96bde09654159/schema/type_descriptions.yaml#L584) for details of the formatting for the `headers_list` field, i believe that captures the requirement here

Reply:
If additional details would be better, i can open an issue in the configuration repo

Reply:
I think it is fine :+1:

Reply:
```suggestion
		// Parsing follows https://github.com/open-telemetry/opentelemetry-configuration/blob/568e5080816d40d75792eb754fc96bde09654159/schema/type_descriptions.yaml#L584.
		headerslist, err := baggage.Parse(*headersList)
```